### PR TITLE
Add StrikerF7

### PR DESCRIPTION
**Pull request by x4FF3**
Hi,

pls add the unified-target for the StrikerF7 board. It will be used for "Soccer Copter".

KR
David

### CODE REVIEW

File: configs/default/AIRB-HELSEL_STRIKERF7.config [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+
+# master
+set baro_bustype = I2C
+set baro_i2c_device = 1
+set current_meter = ADC
+set battery_meter = ADC
+set ibata_scale = 179
+set beeper_inversion = ON
+set beeper_od = OFF
+set pid_process_denom = 2
```
Comment (etracer65) [inline configs/default/AIRB-HELSEL_STRIKERF7.config:87]: There is no reason to set the PID loop to 4K on a F722 board. Just remove this and let it default to the correct setting.

File: configs/default/AIRB-HELSEL_STRIKERF7.config [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+dma pin C09 0
+# pin C09: DMA2 Stream 7 Channel 7
+dma pin B00 0
+# pin B00: DMA1 Stream 7 Channel 5
+dma pin B01 0
+# pin B01: DMA1 Stream 2 Channel 5
+
+# feature
+feature -RX_PARALLEL_PWM
+feature -AIRMODE
```
Comment (etracer65) [inline configs/default/AIRB-HELSEL_STRIKERF7.config:73]: Please don't disable important flight control features in the target config. This should be up to the user.

**Review by mikeller — DISMISSED** [review]
The target as it is is conforming with the rules for Unified Target configurations - but if you want to incorporate the recommendations from @etracer65 feel free to do so.

File: configs/default/AIRB-HELSEL_STRIKERF7.config (hunk 2 above)
Comment (x4FF3) [reply configs/default/AIRB-HELSEL_STRIKERF7.config:73]: It is a stripped version of the customers preset for the soccer copter. As the board will be available as spare part, it is a compromise of the full preset which is configured on the soccer copter by default (filter/PIDs). If the sales show, that the board is bought more than expected, we can change this. But atm we would prefer to leave this like it is.

File: configs/default/AIRB-HELSEL_STRIKERF7.config (hunk 1 above)
Comment (x4FF3) [reply configs/default/AIRB-HELSEL_STRIKERF7.config:87]: I'll remove it.

**Review by mikeller — APPROVED** [review]

**Pull request merged**